C# and .Net/Assignment1/source/repos/PracticeProject1/emailvalidation.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace PracticeProject1
+{
+    class validate
```

Comment:
Pascal case class name. file name and class name should be same for easy understanding.